### ISSUE TITLE
Can not connect Woocommerce to BTCPay Server - Tor

### ISSUE THREAD

**Issue by PsychedelicHell** [btcpayserver/btcpayserver-docker#223]
I am unable to run BTCPay Server via the standard HTTPS way and instead I am trying to use TOR.

When linking the Woocommerce plug-in with BTCPay Server I receive an error "cannot resolve host" with a link to my Tor server (which works when you click on it)

![Can't pair to Woocommerce](https://user-images.githubusercontent.com/57477439/68478834-f9511280-0284-11ea-9062-11a2b1d2f9b7.png)

This is the last step I need to take to get BTC working on my online shop, I have no idea what I can do to fix this, please help.

**Comment by pavlenex** [comment]
I don't think woocommerce plugin works with .onion.

**Comment by PsychedelicHell** [comment]
If that's the case, I am forced to use "[Nomiddleman Crypto Payments for Woocommerce](url)" instead.

**Comment by pavlenex** [comment]
@Kukks can you transfer this issue to woo repository?

**Comment by PsychedelicHell** [comment]
Yes, please create a fix as I have found I also can not use 'Nomiddleman for Woocommerce' because it doesn't support native segwit.

I want to use Tor BTCPay Server with my Woocommerce site.
